### ISSUE TITLE
Segmentation Fault on startup

### ISSUE THREAD

**Issue by riban-bw** [SimulPiscator/aeolus#2]
- Compile [current master version](https://github.com/SimulPiscator/aeolus/commit/ab7c02aa095c0c185af4a8b4511d68cdd6ec2377) (which shows version 0.9.9)
- Install to default location (`make install`)
- Copy stops to /usr/share/aeolus `cp -r stops /usr/share/aeolus/`
- Run aeolus
- Save
- Restart aeolus
- Segmentation fault

It seems to be an attempt to read the default wave files (*.ae0) that triggers the fault. Replacing these with original ones stops the fault but then shows error: "Presets in file '/root/.aeolus-presets' are not compatible". It is as if the ae0 files differ from what the source / preset config expects.

**Comment by riban-bw** [comment]
The definitions file in /usr/share/aeolus/stops/Aeolus/definition is updated by save and this seems to cause the issue. Replacing with the original definition file fixes the fault.

Issue seems to have started with commit ed0e4f3a4b81dd807d8449baebc59d0c619eb0c7.

**Comment by riban-bw** [comment]
Disabling MULTISTOP stops the error so it looks like the multistop stuff added on 2021-02-16 caused this issue. As a work-around I have disabled multistop by default in my local working copy by changing  model.h to set disabled by default:
```
diff --git a/source/model.h b/source/model.h
index 4e4a41a..c3a7c2e 100644
--- a/source/model.h
+++ b/source/model.h
@@ -30,7 +30,7 @@
 #include "global.h"
 
 #ifndef MULTISTOP
-# define MULTISTOP 1
+# define MULTISTOP 0
 #endif
 
 class Asect
```

**Comment by jerash** [comment]
also check the official version of aeolus here : http://kokkinizita.linuxaudio.org/linuxaudio/aeolus/index.html

**Comment by riban-bw** [comment]
Linux-audio shows a 0.10.4 release. Where is the version controlled source code for this? Is SimulPiscator GitHub a fork? It would be good to access and contribute to the official/ upstream code.

**Comment by jerash** [comment]
> Linux-audio shows a 0.10.4 release. Where is the version controlled source code for this? Is SimulPiscator GitHub a fork? It would be good to access and contribute to the official/ upstream code.

yes This is a fork. 0.10.4 is the latest release; there is no official repository, only the page I mentionned. For any question you can ask Fons, or the LAU mailing list where annoncements are made about releases. this is the old way of doing things. You can submit patches on the mailing list.
